src/db/seed.js

```diff
@@ -0,0 +1,68 @@
+// use the class here to refactor
+
+// importing dependencies - DO NOT REMOVE
+const mysql = require("mysql2");
```

Comment:
Try to leave a spare line between any internal and external imports. This seems a bit picky, but when you start using React.js you'll have so many imports, it helps to break them up!